### DISCUSSION TITLE
Bead and Ring Groups Pre-Proposal

### DISCUSSION THREAD

**Discussion by ellabilge** [MDAnalysis/mdanalysis#4511] [GSoC Discussions]
Hello! My name is Ella and I submitted my pre-proposal for bead and ring groups earlier today. I am so excited about this project and looking forward to hearing back about my pre-proposal!

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
